### PR TITLE
Add instructions for verifying Trinity Mobile APKs

### PR DESCRIPTION
**Pull request by rajivshah3**
# Description of change

For advanced users that do not use the Play Store, we will begin offering the ability to directly download the Trinity APK from GitHub. This PR includes instructions for verifying the SHA-256 hash and code signature of the APK to verify it is legitimate. This documentation is based on similar [instructions](https://docs.iota.org/docs/wallets/0.1/trinity/how-to-guides/verify-trinity-desktop) for Trinity Desktop.

# Type of change

- Enhancement (a non-breaking change which adds new content)

# Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my changes

### CODE REVIEW

**Comment by rajivshah3** [comment]
Requesting additional review from @cvarley100 and @nothingismagick

File: wallets/0.1/trinity/how-to-guides/verify-trinity-mobile-android.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+:::warning:
+Installing Android apps, or APKs, from sources outside of the Play Store is dangerous. It is much easier to impersonate legitimate apps when they are distributed outside of the Play Store. When in doubt, you should download Trinity from the [Play Store](https://play.google.com/store/apps/details?id=com.iota.trinity). The following instructions are for advanced users that are comfortable with the increased risk of downloading APKs from sources other than the Play Store.
+:::
+
+:::danger:
+On 11 February 2020, the IOTA Foundation became aware of an attack on the Trinity wallet, during which some users’ seeds and Trinity passwords were compromised. Please check our advice for [protecting your Trinity account](../how-to-guides/protect-trinity-account.md).
+:::
+
+To verify the authenticity of Trinity Mobile for Android, you can check its SHA256 hash and its code signature. Instructions for both of these steps differ--verification of the SHA256 hash can be done from your Android device, but a computer is required in order to verify the code signature.
```
Comment (JakeSCahill) [inline wallets/0.1/trinity/how-to-guides/verify-trinity-mobile-android.md:13]: I think as this is a single topic, the prerequisites should be a level up.

I'd suggest:
```
To verify the authenticity of Trinity Mobile for Android, you can check its SHA256 hash and its code signature.

## Prerequisites

To verify the SHA256 hash, you can use your Android device.

To verify the code signature, you'll need a device with the [Java Development Kit (JDK)](https://www.oracle.com/java/technologies/javase-downloads.html).
```

File: wallets/0.1/trinity/how-to-guides/verify-trinity-mobile-android.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+2. Select "SHA256" for "Hash algorithm".
+
+3. Select the Trinity APK and upload it.
+
+4. Compare your SHA256 hash with the one in the [release notes](https://github.com/iotaledger/trinity-wallet/releases) and make sure that they match
+
+### Verify the code signature
+
+#### Prerequisites
```
Comment (JakeSCahill) [inline wallets/0.1/trinity/how-to-guides/verify-trinity-mobile-android.md:27]: Remove this part from here

File: wallets/0.1/trinity/how-to-guides/verify-trinity-mobile-android.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+Installing Android apps, or APKs, from sources outside of the Play Store is dangerous. It is much easier to impersonate legitimate apps when they are distributed outside of the Play Store. When in doubt, you should download Trinity from the [Play Store](https://play.google.com/store/apps/details?id=com.iota.trinity). The following instructions are for advanced users that are comfortable with the increased risk of downloading APKs from sources other than the Play Store.
+:::
+
+:::danger:
+On 11 February 2020, the IOTA Foundation became aware of an attack on the Trinity wallet, during which some users’ seeds and Trinity passwords were compromised. Please check our advice for [protecting your Trinity account](../how-to-guides/protect-trinity-account.md).
+:::
+
+To verify the authenticity of Trinity Mobile for Android, you can check its SHA256 hash and its code signature. Instructions for both of these steps differ--verification of the SHA256 hash can be done from your Android device, but a computer is required in order to verify the code signature.
+
+### Verify the SHA256 hash
```
Comment (JakeSCahill) [inline wallets/0.1/trinity/how-to-guides/verify-trinity-mobile-android.md:15]: Second level `##`

File: wallets/0.1/trinity/how-to-guides/verify-trinity-mobile-android.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+1. Go to https://www.sisik.eu/hash.
+
+2. Select "SHA256" for "Hash algorithm".
+
+3. Select the Trinity APK and upload it.
+
+4. Compare your SHA256 hash with the one in the [release notes](https://github.com/iotaledger/trinity-wallet/releases) and make sure that they match
+
+### Verify the code signature
```
Comment (JakeSCahill) [inline wallets/0.1/trinity/how-to-guides/verify-trinity-mobile-android.md:25]: Second level `##`

File: wallets/0.1/trinity/how-to-guides/verify-trinity-mobile-android.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+#### Prerequisites
+To follow these instructions you will need a computer with [Java Development Kit (JDK)](https://www.oracle.com/java/technologies/javase-downloads.html) installed.
+
+1. Download the Trinity APK to your computer
+
+2. Open a command-line interface such as Terminal or PowerShell.
+
+3. Verify the signature of the Trinity APK, replacing the path with the path to your Trinity APK on your computer:
+```bash
```
Comment (JakeSCahill) [inline wallets/0.1/trinity/how-to-guides/verify-trinity-mobile-android.md:35]: This needs to be indented (4 spaces) so our markdown parser will format it correctly.

File: wallets/0.1/trinity/how-to-guides/verify-trinity-mobile-android.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+keytool -printcert -jarfile path/to/trinity.apk
+```
+
+For example, if the APK is in `~/Downloads`:
```
Comment (JakeSCahill) [inline wallets/0.1/trinity/how-to-guides/verify-trinity-mobile-android.md:39]: Same with this part

**Comment by rajivshah3** [comment]
Thanks @JakeSCahill , will make the appropriate changes

> Will we have instructions somewhere so users know where and how to download the APK?

Yep, the link will be in the release notes on GitHub

**Comment by rajivshah3** [comment]
@nothingismagick I agree, we should follow the same procedure as Trinity Desktop (SHA-256 in the release notes). Not sure if it needs to be tweeted and posted on the IOTA website though 🤔

**Pull request merged**